digiroad2-geo/src/test/scala/fi/liikennevirasto/digiroad2/linearasset/LinearAssetPartitionerSpec.scala

```diff
@@ -4,11 +4,17 @@ import fi.liikennevirasto.digiroad2.Point
 import fi.liikennevirasto.digiroad2.asset._
 import org.joda.time.DateTime
 import org.scalatest._
+import java.util.UUID
+import scala.util.Random
 
 class LinearAssetPartitionerSpec extends FunSuite with Matchers {
   case class TestLinearAsset(id: Long, linkId: String, sideCode: SideCode, value: Option[NumericValue], geometry: Seq[Point], vvhTimeStamp: Long = 0, geomModifiedDate: Option[DateTime] = None) extends LinearAsset
   case class TestProhibitionAsset(id: Long, linkId: String, sideCode: SideCode, value: Option[Prohibitions], geometry: Seq[Point], vvhTimeStamp: Long = 0, geomModifiedDate: Option[DateTime] = None) extends LinearAsset
 
+  private def generateRandomLinkId(): String = s"${UUID.randomUUID()}:${Random.nextInt(100)}"
```

Comment:
onkos tässä duplijaattu koodia ton uuden gerenerator method kanssa

Reply:
tais tulla build.scala module ongelma

Reply:
pitäis kyl ehkä luopuo koko module hommasta kun ei me tehdä omaa jar per module

Reply:
Joo se tässä oli ongelmana kun geo jar ei oo riippuvainen mistään muusta moduulista. Oracle jar, jossa id generointi funktion on, taas riippuu geo jar:ista. Sinänsä tässä ei hirveän paljoa duplikaatti koodia tule kun saadaan id generoitua yhdellä rivillä ja testi tiedostoja ei ole montaa geo puolella.